app/js/components/encounters/observations.jsx

```diff
@@ -41,136 +41,101 @@ class Observations extends React.Component {
-                  <th>Value</th>
-                  <th>Created</th>
-                </tr>
-              </thead>
-              <tbody>
-                {
-                  unVoidedObs && unVoidedObs.length > 0 &&
-                  unVoidedObs.map((ob) => {
-                    if (ob.groupMembers !== null) {
-                      return (
-                        (ob.groupMembers.map(observation => (
-                          <tr key={observation.uuid}>
-                            <a>
-                              <td
-                                onClick={() => { this.props.handleObservationClick(observation.uuid); }}
-                              >{observation.concept.display}
-                              </td>
-                            </a>
-                            <td>{observation.value.display}</td>
-                            <td>{new Date(observation.obsDatetime).toString()}</td>
-                          </tr>
-                        )))
-                      );
-                    }
-                    return (
-                      <tr>
-                        <a>
-                          <td
-                            onClick={() => { this.props.handleObservationClick(ob.uuid); }}
-                          >{ob.concept.display}
-                          </td>
-                        </a>
-                        <td>{ob.value}</td>
-                        <td>{new Date(ob.obsDatetime).toString()}</td>
-                      </tr>
-                    );
-                  })
-                }
-              </tbody>
-            </table>
-          </TabPane>
-          <TabPane tabId="2">
-            <table className="table table-striped">
-              <thead>
-                <tr>
-                  <th>Question Concept</th>
-                  <th>Value</th>
-                  <th>Created</th>
-                </tr>
-              </thead>
-              <tbody>
-                {
-                  voidedObs && voidedObs.length > 0 &&
-                  voidedObs.map((ob) => {
-                    if (ob.groupMembers !== null) {
-                      return (
-                        (ob.groupMembers.map(observation => (
-                          <tr key={observation.uuid}>
-                            <a>
-                              <td
-                                onClick={() => { this.props.handleObservationClick(observation.uuid); }}
-                              >{observation.concept.display}
-                              </td>
-                            </a>
-                            <td>{observation.value.display}</td>
-                            <td>{new Date(observation.obsDatetime).toString()}</td>
+        </header>
+
+        <span id="show-deleted">
+          <input
+            name="voided"
+            className="form-check-input"
+            type="checkbox"
+            value=""
+            onChange=""
+          /> Show Deleted
+        </span>
```

Comment:
If the value and onChange input props are not being set to anything, do we need to have them mentioned?

Reply:
@jacksono  good catch. I meant to use these two during display of soft deleted observations but for that functionality a bug in web service needs to be resolved first. For now I'll remove them. Thanks